### ISSUE TITLE
Beautify inserts space before exclamation mark in comment <!-- in <style>

### ISSUE THREAD

**Issue by Codebitt** [brackets-beautify/brackets-beautify#278]
**Details about your environment:**
* Brackets Version: windows-release-1.13.1 fc2ebbeea
* Brackets Beautify Version: 2.7.0

Beautify is adding space before exclamation mark followed by opening tag between <style>&</style>.

**What I mean:**
```
<style type="text/css"><!--
--></style>
```
**What happens:**
```
<style type="text/css">< !--
--></style>
```
**What should happen:**
```
<style type="text/css"><!--
--></style>
```

**Comment by bitwiseman** [comment]
Ah, yes.  This isn't new bug. The javascript beautifier handles this, but the css does not.  

Questions:
Do you have an webpages the talk about why one would do this?  
Also, is this only valid as an outer wrapper for the contents of the script tag or could you put html comments like this in the middle of the script?  

I ask because if it is only a wrapper, I could move the functionality into the html beautifier and this would be handled for all script/style languages. 

I've opened https://github.com/beautify-web/js-beautify/issues/1641 to track this.

**Comment by Hirse** [comment]
Fixed in js-beautify 1.10.0 and brackets-beautify 2.9.0
